File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Concert/ConcertInsertConcertDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ArenaGestor.APIContracts.Concert
{
    public class ConcertInsertConcertDto
    {
        [MaxLength(50)]
        [Required]
        public string TourName { get; set; }
        [Required]
        public DateTime Date { get; set; }
        [Required]
        public int TicketCount { get; set; }
        [Required]
        public double Price { get; set; }
        [Required]
        public List<ConcertInsertProtagonistDto> Protagonists { get; set; }
        [Required]
        public ConcertInsertLocationDto Location { get; set; }
    }
}

File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Concerts/ConcertInsertConcertDto.ts
import { ConcertInsertLocationDto } from "./ConcertInsertLocationDto";
import { ConcertInsertProtagonistDto } from "./ConcertInsertProtagonistDto";

export class ConcertInsertConcertDto {
    tourName: String = "";
    date: Date = new Date();
    ticketCount: Number = 0;
    price: Number = 0;
    protagonists: Array<ConcertInsertProtagonistDto> = new Array();
    location: ConcertInsertLocationDto = new ConcertInsertLocationDto();
}